datadog_checks_downloader/datadog_checks/downloader/download.py

```diff
@@ -292,7 +276,7 @@ def __get_versions(self, standard_distribution_name):
         try:
             # NOTE: We do not perform in-toto inspection for simple indices; only for wheels.
             index_abspath, _ = self.__download_with_tuf(index_relpath)
-        except UnknownTargetError:
+        except TargetNotFoundError:
```

Comment:
I think we do, I don't think it would make sense for a `__download_with_tuf` method that's used to generically download artifacts from the repository to raise a `NoSuchDatadogPackage` exception. The two exceptions are at different abstraction levels and communicate different things.